marklogic-data-hub-central/ui/src/components/load/new-load-dialog/new-load-dialog.test.tsx

```diff
@@ -102,4 +102,29 @@ describe('New/edit load data configuration', () => {
     expect(queryByText('Source Type')).not.toBeInTheDocument();
   });
 
+  test('Verify clicking "Save" with no name shows error', () => {
```

Comment:
Can we check also check that when content is entered in the name field, the error messaging goes away? You could update the test description to something like: "Verify name is required for form submission and error messaging appears"

Reply:
added